README.md

```diff
@@ -6,14 +6,15 @@
 [![Join our tinyletter mailing list](https://img.shields.io/badge/receive-our%20newsletter%20❤%EF%B8%8F-blueviolet.svg)](https://tinyletter.com/TuringWay)
 [![Join the chat at https://gitter.im/alan-turing-institute/the-turing-way](https://img.shields.io/gitter/room/alan-turing-institute/the-turing-way?logo=gitter)](https://gitter.im/alan-turing-institute/the-turing-way)
 
-The Turing Way is a lightly opinionated guide to reproducible data science.
+The Turing Way is a lightly opinionated guide to reproducible data science. You can read it here: https://the-turing-way.netlify.com. This is the GitHub repository where all of the bits that make up the guide live, and where the process of writing/building the guide happens.
```

Comment:
```suggestion
The Turing Way is a lightly opinionated guide to reproducible data science.
You can read it here: https://the-turing-way.netlify.com.
You're currently viewing the project GitHub repository where all of the bits that make up the guide live, and where the process of writing/building the guide happens.
```

Reply:
Hi @da5nsy - I think you clicked "resolve" rather than commit suggestion - do you mind accepting the suggestion (click commit suggestion button) so we have each sentence on separate lines?